tests/framework/PostProcessors/Validation/test_validation_dss.xml

```diff
@@ -3,19 +3,19 @@
   <TestInfo>
     <name>framework/PostProcessors/Validation/test_validation_dss</name>
     <author>yoshrk</author>
-    <created>2021-03-16</created>
+    <created>2022-05-13</created>
```

Comment:
You may not need to change the date, but add additional <revision> node in <revisions>

Reply:
Added new revision node.